### DISCUSSION TITLE
Propose https://github.com/zeotuan as a new maintainer

### DISCUSSION THREAD

**Discussion by SemyonSinchenko** [mrpowers-io/spark-fast-tests#148] [General]
I would like to propose [zeotuan](https://github.com/zeotuan) as a new maintainer of the project.
Contributions from zeotuan: https://github.com/mrpowers-io/spark-fast-tests/commits?author=zeotuan

Please vote: +1, +0, -1 (with a comment)
Votes from @alfonsorr, @SemyonSinchenko and @MrPowers are binding, other votes are non-binding.

**Comment by SemyonSinchenko** [comment]
+1 (binding)

**Comment by MrPowers** [comment]
+1 (binding)

**Comment by alfonsorr** [comment]
+1

**Comment by SemyonSinchenko** [comment]
Results:
3 +1 (3 binding)
0 +0
0 -1

Vote is passed;
@zeotuan welcome to the project in a new role!
